fcrepo-service-ldcache-file/src/main/resources/OSGI-INF/blueprint/ldcache-service.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No, for our purposes, it just needs to have an xml extension and be in the `OSGI-INF` directory.